docker/platypus-deps/Dockerfile

```diff
@@ -173,4 +187,13 @@ WORKDIR /$WORKDIR/moose/framework
 RUN METHOD=opt make -j$compile_cores
 
 WORKDIR /$WORKDIR/moose/test
-RUN METHOD=opt make -j$compile_cores
\ No newline at end of file
+RUN METHOD=opt make -j$compile_cores
+
+# Build HEAT_TRANSFER module 
+WORKDIR /$WORKDIR/moose/modules/heat_transfer
+RUN METHOD=opt make -j$compile_cores
+
+# Build SOLID_MECHANICS module 
+WORKDIR /$WORKDIR/moose/modules/solid_mechanics
+RUN METHOD=opt make -j$compile_cores
```

Comment:
Rather than this - it's probably better to just change the `WORKDIR` to `/$WORKDIR/moose/modules` and then run the make command with options, eg:
`METHOD=opt make ALL_MODULES=no HEAT_TRANSFER=yes SOLID_MECHANICS=yes`

Some modules require other modules to be built, and the above will be more tolerant in cases when MOOSE changes module names, since old module names tend to be supported in the makefile (as deprecated names) much longer than the time it takes for the location to be updated upstream.